popup_windows.js

```diff
@@ -0,0 +1,133 @@
+
+  html: 'html',
+  css: 'css',
+  ror: 'Ruby on Rails',
+  github: 'Github',
+};
+
+/* Basic script */
+
+closeProject.innerHTML = '&times;';
+closeProject.classList.add('close-project');
+
+const h1 = document.createElement('h1');
+h1.innerHTML = 'Multi - Post stories';
+h1.classList.add('title');
+
+title.classList.add('popup-title');
+title.appendChild(h1);
+title.appendChild(closeProject);
+
+const popupImage = document.createElement('img');
+popupImage.classList.add('popup-image');
+
+const popupText = document.createElement('p');
+popupText.classList.add('popup-text');
+
+const technologie1 = document.createElement('li');
+technologie1.innerHTML = text.html;
+technologie1.classList.add('tech', 'popup-tech');
+
+const technologie2 = document.createElement('li');
+technologie2.innerHTML = text.ror;
+technologie2.classList.add('tech', 'popup-tech');
+const technologie3 = document.createElement('li');
+
+technologie3.innerHTML = text.css;
+technologie3.classList.add('tech', 'popup-tech');
+const technologie4 = document.createElement('li');
+
+technologie4.innerHTML = text.github;
+technologie4.classList.add('tech', 'popup-tech');
+technologie4.style.borderRight = '2px solid #646369';
+
+const technologies = document.createElement('ul');
+technologies.classList.add('techs', 'popup-techs');
+technologies.appendChild(technologie1);
+technologies.appendChild(technologie2);
+technologies.appendChild(technologie3);
+technologies.appendChild(technologie4);
+
+const seeLive = document.createElement('button');
+seeLive.innerHTML = 'See live';
+seeLive.classList.add('see-project', 'popup-button');
+const live = document.createElement('img');
+live.setAttribute('src', './images/buttons/Icon_live.svg');
+seeLive.appendChild(live);
+
+const seeSource = document.createElement('button');
+seeSource.innerHTML = 'See source';
+seeSource.classList.add('see-project', 'popup-button');
+const source = document.createElement('img');
+source.setAttribute('src', './images/buttons/github_white.svg');
+seeSource.appendChild(source);
+
+const buttons = document.createElement('div');
+buttons.classList.add('popup-buttons');
+buttons.appendChild(seeLive);
+buttons.appendChild(seeSource);
+
+function mediaqueries(e) {
+  if (e.matches) { // If media query matches
+    technologie4.style.display = 'none';
+    popupText.innerHTML = text.txtMobile;
+    popupImage.setAttribute('src', image.imgMobile);
+    h1.style.fontSize = '20px';
+  } else {
+    h1.style.fontSize = '40px';
+    technologie4.style.display = 'flex';
+    popupText.innerHTML = text.txtDesktop;
+    popupImage.setAttribute('src', image.imgDesktop);
+  }
+}
+
+const media = window.matchMedia("(max-width: 600px)");
+mediaqueries(media);
+media.addListener(mediaqueries);
+
+container.classList.add('project-card');
+container.appendChild(title);
+container.appendChild(popupImage);
+container.appendChild(popupText);
+container.appendChild(technologies);
+container.appendChild(buttons);
```

Comment:
- [ ] The JavaScript code implements an array of objects to store JavaScript objects containing the information for each project. (Currently, you are only appending the child elements to your containers without creating an array of objects which contain the necessary information per object).